include/aikido/planner/parabolic/ParabolicSmoother.hpp

```diff
@@ -129,6 +130,60 @@ std::unique_ptr<trajectory::Spline> doShortcutAndBlend(
     double _checkResolution = DEFAULT_CHECK_RESOLUTION,
     double _tolerance = DEFAULT_TOLERANCE);
 
+/// Class for performing parabolic smoothing on trajectories
+class SmoothTrajectoryPostProcessor
```

Comment:
I'd still like to rename this `ParabolicSmoother` (and I think it makes more sense now that it's in this file).